code/__DEFINES/combat.dm

```diff
@@ -56,7 +54,6 @@
 #define BRUTELOSS (1<<0)
 #define FIRELOSS (1<<1)
 #define TOXLOSS (1<<2)
-#define CLONELOSS (1<<3)
 #define OXYLOSS (1<<4)
 #define STAMINALOSS (1<<5)
 #define SHAME (1<<6)
```

Comment:
you need to shift all the bitflags past where cloneloss was down by one, cant' have it empty